openmls/src/group/tests/test_external_commit_validation.rs

```diff
@@ -528,24 +558,39 @@ fn test_valsem244(ciphersuite: Ciphersuite, backend: &impl OpenMlsCryptoProvider
 
     // Bob wants to commit externally.
 
-    // Have Alice export everything that bob needs.
-    let pgs_encoded: Vec<u8> = alice_group
-        .export_public_group_state(backend)
-        .expect("Error exporting PGS")
-        .tls_serialize_detached()
-        .expect("Error serializing PGS");
-    let verifiable_public_group_state =
-        VerifiablePublicGroupState::tls_deserialize(&mut pgs_encoded.as_slice())
-            .expect("Error deserializing PGS");
+    let group_info = {
+        // Create to-be-signed group info.
+        let group_info_tbs = GroupInfoTBS::new(
+            alice_group.export_group_context().clone(),
+            alice_group.group().other_extensions().as_slice(),
+            ConfirmationTag(Mac {
```

Comment:
I'm not sure what should happen here.

Reply:
You can put in anything here because the joiner can't check the tag.